src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,351 @@
+                          link.subnav.items.map((link) => (
+                            <div
+                              key={link.id}
+                              className={styles.accordionChild}
+                            >
+                              <TitleSmall.Sans.Light300>
+                                <NavLink
+                                  href={link.href}
+                                  LinkComponent={
+                                    link.href !== '/login/'
+                                      ? LinkComponent
+                                      : null
+                                  }
+                                >
+                                  {link.title}
+                                </NavLink>
+                              </TitleSmall.Sans.Light300>
+                            </div>
+                          ))}
+                      </div>
+                    </div>
+                  ))}
+                </div>
+                <div className={styles.belowAccordion}>
+                  <div className={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.TERM.href}
+                      LinkComponent={
+                        LINKS.TERM.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.TERM.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                  <div className={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.ACCOUNT.href}
+                      LinkComponent={
+                        LINKS.ACCOUNT.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.ACCOUNT.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                  <div className={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.SEARCH.href}
+                      LinkComponent={
+                        LINKS.SEARCH.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.SEARCH.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                </div>
+              </div>
+
+              {/* Mobile menu items, getAnEstimate only shows when scrolled */}
+              <NavLink
+                href={LINKS.INDEX.href}
+                LinkComponent={LinkComponent}
+                className={styles.phoneLogoFancy}
+              >
+                {FancyAnimatedLogo()}
```

Comment:
Does this need to be invoked? It can be a functional component right

Reply:
this is from the old version, i'll see about updating

Reply:
marked in TODOs for fastfollow